package.json

```diff
@@ -86,7 +86,7 @@
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.9.2",
     "webpack-merge": "^5.8.0",
-    "yarn": "^1.22.13"
+    "yarn": "^1.22.18"
```

Comment:
This change still needs backing out I think. It shouldn't be needed here, and undoing this should also fix the CI run.